packages/core/src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -17,6 +17,17 @@
             "allowedValues": ["Create", "Update"],
             "description": "SSM Publish Document operation type"
         },
+        {
+            "name": "stepFunctionsSyncType",
+            "type": "string",
+            "allowedValues": ["MANUAL_SAVE_WFS", "MANUAL_SAVE_VSCODE", "AUTO_SYNC_WFS"],
```

Comment:
I used it this way because the previous version of the metric that I was using (`saveType` introduced by Threat Composer integration) had the same screaming snake casing.

I can update the metric to camel case to keep this consistent with most of the types